analyses/optimus/create_synthetic_data.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+# this script takes as input the PBMC8k dataset, multi-aligned with Optimus.
+# it generates a bam file containing unique, unaligned, and multialigned reads from different
+# chromosomes
+# it then regenerates fastq data
+
+set -eo pipefail
+
+# grab the picard jar
+curl -o picard.jar https://github.com/broadinstitute/picard/releases/download/2.18.4/picard.jar
```

Comment:
Who is going to run this script and how often? Depending on the context, it might be nice to write a Dockerfile that includes the picard jar, samtools, and this script, plus a script that will run the script in the container and connect things up to the local file system appropriately.

Reply:
A CB will run this script whenever synthetic data is needed from a new dataset. I suspect it will not be frequent. A docker could be useful and in absence of time constraints I think it would be a good idea. 

However, given bandwidth constraints it might be good to leave an issue and punt on this for now.